pkgs/applications/emulators/cdemu/analyzer.nix

```diff
@@ -1,20 +1,14 @@
-{ callPackage, makeWrapper, gobject-introspection, cmake
+{ callPackage, makeWrapper, gobject-introspection, cmake, wrapGAppsHook3
 , python3Packages, gtk3, glib, libxml2, gnuplot, gnome, gdk-pixbuf, librsvg, intltool, libmirage }:
-let pkg = import ./base.nix {
-  version = "3.2.5";
+callPackage ./base.nix {
+  version = "3.2.6";
```

Comment:
Probably best to do cleanup first and then update in a separate commit so that it is easier to review.